generic-worker.Dockerfile

```diff
@@ -32,11 +33,18 @@ RUN mkdir -p /var/local/generic-worker
 # autogenerated ed25519 key is only good for local and testing
 RUN generic-worker new-ed25519-keypair --file /etc/generic-worker/ed25519_key
 
+# Write out the DockerFlow-compatible version.json file
+ARG DOCKER_FLOW_VERSION
+RUN if [ -n "${DOCKER_FLOW_VERSION}" ]; then \
+    echo "${DOCKER_FLOW_VERSION}" > /version.json; \
+else \
+    echo \{\"version\": \"44.18.0\", \"commit\": \"local\", \"source\": \"https://github.com/taskcluster/taskcluster\", \"build\": \"NONE\"\} > /version.json; \
```

Comment:
Does this need to be patched to 44.19.1 now so that the regex can update it?

Reply:
yes, thanks! Added it also to the release tasks to upgrade version here, too.